pkgs/development/python-modules/platformdirs/default.nix

```diff
@@ -1,32 +1,29 @@
 { lib
 , appdirs
 , buildPythonPackage
-, fetchFromGitHub
-, platformdirs
+, fetchPypi
 , pytest-mock
 , pytestCheckHook
 , pythonOlder
-, setuptools-scm
+, hatchling
+, hatch-vcs
 }:
 
 buildPythonPackage rec {
   pname = "platformdirs";
-  version = "2.5.1";
-  format = "setuptools";
+  version = "2.5.2";
+  format = "pyproject";
 
   disabled = pythonOlder "3.7";
 
-  src = fetchFromGitHub {
-    owner = pname;
-    repo = pname;
-    rev = version;
-    sha256 = "sha256-z6WIwTWLlc/chNRxt3dqqa/IxYj1BBTcQ6OcfliHrvA=";
+  src = fetchPypi {
```

Comment:
Why do we need to switch to use the source from PyPI?

Reply:
Because the new `hatchling` build system requires version metadata which isn't available from a source clone.

Reply:
Based on dependencies, hatch-vcs appears to wrap setuptools-scm, I'm wondering if the version pretend variable might still continue to work, or maybe the variable is called differently?

Reply:
Yep, this works! Thanks.